Public/Src/Engine/Scheduler/Tracing/ExecutionLog.Events.cs

```diff
@@ -1280,6 +1285,7 @@ public void Serialize(BinaryLogger.EventWriter writer)
             writer.Write(LookupRunning);
 
             writer.Write(ExternalProcesses);

```

Comment:
We might rename ExternalProcesses to PipExecutorProcesses. your call?

Reply:
sounds good